.github/workflows/scan-lint.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Run Revive Action by pulling pre-built image
-        uses: docker://morphy/revive-action:v2
+        uses: docker://morphy/revive-action@sha256:087d4e61077087755711ab7e9fae3cc899b7bb07ff8f6a30c3dfb240b1620ae8 # v2.5.7
```

Comment:
does this get picked up by renovate?

Reply:
Yes

https://docs.renovatebot.com/modules/manager/github-actions/#file-matching

> By default, Renovate will check any files matching any of the following regular expressions:

```plaintext
(^|/)(workflow-templates|\.(?:github|gitea|forgejo)/(?:workflows|actions))/.+\.ya?ml$
(^|/)action\.ya?ml$
```

https://docs.renovatebot.com/modules/manager/github-actions/#additional-information

> The github-actions manager extracts dependencies from GitHub Actions workflow and workflow template files. It can also be used for Gitea and Forgejo Actions workflows as such are compatible with GitHub Actions workflows.

> If you like to use digest pinning but want to follow the action version tag, you can use the sample below:

```yaml
name: build

on: [push]

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
```

Reply:
https://github.com/renovatebot/renovate/issues/25276

Reply:
sgtm